common/src/main/java/com/radixdlt/networks/FixedNetworkGenesis.java

```diff
@@ -64,13 +64,16 @@
 
 package com.radixdlt.networks;
 
-public sealed interface GenesisSource {
-  FromConfiguration fromConfiguration = new FromConfiguration();
-  AsynchronouslyProvided providedAsync = new AsynchronouslyProvided();
+public sealed interface FixedNetworkGenesis {
+  record Resource(String resourcePath) implements FixedNetworkGenesis {}
```

Comment:
I was imagining that we'll ship the genesis file in the `resources` dir, hence `Resource` here.

Reply:
Ahh I see....

We possibly will at some point. But what about for just-after-launch? I imagine we'll want to provide a small genesis file to download; rather than requesting people reinstall the whole node? Or can you add a file to resources very easily?

Reply:
Ah, gotcha. That can still be done with `network.genesis_file` as long as it matches the genesis assigned to a given `Network`, or if it's empty - and that'll be the case at launch.